virttest/qemu_devices/qdevices.py

```diff
@@ -920,6 +920,11 @@ def __init__(self, aobject):
         super(QBlockdevProtocol, self).__init__(aobject, None, False)
 
 
+class QBlockdevProtocolVirtioBlkVhostVdpa(QBlockdevProtocol):
```

Comment:
Comments for commit header and body:
qdevices: Add a protocol node for virtio-blk-vhost-vdpa

Add a new class: QBlockdevProtocolVirtioBlkVhostVdpa

or
qdevices: Add QBlockdevProtocolVirtioBlkVhostVdpa for virtio-blk-vhost-vdpa (this might be too long)